src/assets/icons/icon-trash-dark.svg

```diff
@@ -0,0 +1,6 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" viewBox="0 0 16 16">
+  <g fill="#32363f" fill-rule="evenodd" transform="translate(-96 -32)">
```

Comment:
I'm too questioning that choice more and more lately. It's semantically more correct to use the background technique but not really the more pragmatic way. Any thought @m4dz?